### PR TITLE
GET /v1/lenses/:key should have a "user" attribute when returnUser toggle is on

### PR DESCRIPTION
**Pull request by annyhe**
- with get.js tests
- add tests on /v1/lenses methods (POST, DELETE, etc.) to show the user is returned

### CODE REVIEW

File: tests/api/v1/lenses/delete.js [hunk 1]
```diff
@@ -22,53 +22,93 @@ const ZERO = 0;
...
+        expect(res.body.user).to.be.an('object');
+        expect(res.body.installedBy).to.equal(userId);
+        done();
+      });
+    });
   });
 
-  afterEach(u.forceDelete);
-  after(tu.forceDeleteUser);
+  describe('with returnUser toggle off, user object should be returned: ', () => {
```
Comment (jgraff2) [inline tests/api/v1/lenses/delete.js]: "user object should not be returned"

File: tests/api/v1/lenses/delete.js [hunk 2]
```diff
@@ -22,53 +22,93 @@ const ZERO = 0;
...
+      .expect(constants.httpStatus.OK)
+      .end((err, res) => {
+        if (err) {
+          return done(err);
+        }
 
-      expect(res.body.isDeleted).to.not.equal(ZERO);
-      done();
+        expect(res.body.isDeleted).to.not.equal(ZERO);
+        expect(res.body.installedBy).to.not.be.defined;
```
Comment (jgraff2) [inline tests/api/v1/lenses/delete.js]: is this supposed to be res.body.user?

File: tests/api/v1/lenses/delete.js (hunk 1 above)
Comment (annyhe) [reply tests/api/v1/lenses/delete.js]: yes will update

File: tests/api/v1/lenses/delete.js (hunk 2 above)
Comment (annyhe) [reply tests/api/v1/lenses/delete.js]: yes will update

**Pull request merged**